ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -94,17 +94,52 @@ protected override BorderedCanvas CreateViewInstance(ThemedReactContext reactCon
 
         /// <summary>
         /// Sets whether or not the view is an accessibility element.
+        /// Since "accessible" is an Android-only ReactNative property and also is not in ReactXP at all,
+        /// this method is noop and exists just to avoid crashes if for some reason the property
+        /// gets to this level. For similar functionality see <see cref="SetImportantForAccessibility"/>
         /// </summary>
         /// <param name="view">The view.</param>
         /// <param name="accessible">A flag indicating whether or not the view is an accessibility element.</param>
         [ReactProp("accessible")]
         public void SetAccessible(BorderedCanvas view, bool accessible)
         {
-            // TODO: #557 Provide implementation for View's accessible prop
+            // This space is intentionally left blank.
+        }
 
-            // We need to have this stub for this prop so that Views which
-            // specify the accessible prop aren't considered to be layout-only.
-            // The proper implementation is still to be determined.
+        /// <summary>
+        /// Set accessibility traits for the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="accessibilityTraitsValue">Can be <see cref="JArray"/> of objects or a single object. 
+        ///     String representation of the object(s) is parsed as <see cref="AccessibilityTrait"/>.</param>
+        [ReactProp("accessibilityTraits")]
+        public void SetAccessibilityTraits(BorderedCanvas view, object accessibilityTraitsValue)
+        {
+            AccessibilityTrait[] result = null;
+            if (accessibilityTraitsValue != null)
+            {
+                if (accessibilityTraitsValue is JArray asJArray)
```

Comment:
>accessibilityTraitsValue is JArray asJArray [](start = 20, length = 43)

this notation creates a variable called asJArray?

Reply:
yes, in case the pattern matches (is) the asJArray variable will be available and definitely initialized. It's a new C# 7 feautre [MSDN](https://docs.microsoft.com/en-us/dotnet/csharp/pattern-matching)